.github/workflows/pr-test.yml

```diff
@@ -1,10 +1,10 @@
+# TODO - different name since if its not going to run on PRs
```

Comment:
This workflow requires access to secrets for `ROLE_TO_ASSUME` which open source contributors outside the org will not have access to which results in the error

```
Error: Credentials could not be loaded, please check your action inputs: Could not load credentials from any providers
```

I think most of what happens in this workflow is duplicated in https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/main/.github/workflows/pre-commit.yaml, we could probably remove this and rely on the other tests we have in place. TBD